recipe/meta.yaml

```
@@ -1,28 +1,32 @@
 {% set name = "google-cloud-vision" %}
-{% set version = "3.3.1" %}
+{% set version = "3.8.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/google-cloud-vision-{{ version }}.tar.gz
-  sha256: 1bbbf4f42035a49d8e48bee007362f7d11ab8cf8d0125c67c9e52f3f6be91494
+  url: https://files.pythonhosted.org/packages/c3/fe/32f354e0a356b8972819944ebf226970300121d6468f6a02b4e02e4d262e/google_cloud_vision-3.8.1.tar.gz
+  sha256: 75c7f9c22199ba663cea7d6c1193d5a72ac1e5e2215346a852af133ce69c4f5f
```

Stick to the original URL scheme as it means PyPI can change the physical pathnames under our feet and we aren't affected.

The reason the URL wasn't found is the (common) trick of replacing `-`s with `_`s, as in `google-cloud-vision` -> `google_cloud_vision`

```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/google_cloud_vision-{{ version }}.tar.gz
  sha256: ... 
```